MBLD scrambling guidelines

Rouxles opened the pull request.

Rouxles committed:
Update wca-guidelines.md

Nanush7 commented on wca-guidelines.md:
```diff
@@ -93,6 +93,7 @@ To be more informative, each Guideline is classified using one of the following
 - 4d++) [ADDITION] It is permitted for the puzzle to change its orientation when it is moved from the scrambler to the solving station, as long as no one is attempting to influence the randomness of the orientation (see [Regulation A2e1](regulations:regulation:A2e1)).
 - 4f+) [RECOMMENDATION] The WCA Delegate should generate sufficient scramble sequences for the entire competition ahead of time, including spare scramble sequences for extra attempts.
 - 4f++) [REMINDER] If the WCA Delegate generates any additional scramble sequences during the competition, the scramble sequences must be saved.
+- 4g+) [ADDITION] For Multi-Blind Solving, the scrambler should verify that the puzzles are scrambled correctly and that there are no duplicate scrambles.
```
```suggestion
- 4g+) [RECOMMENDATION] For Multi-Blind Solving, the scrambler should pay special attention to duplicate scrambles.
```

I think it's better to remove that part, so we don't give the idea that checking the scrambles is a "should". 4g already states that they must do it.

CarterKoala committed:
Update wca-guidelines.md

I think you're right, but I prefer the wording in the original change (verify there are no duplicates) as your wording doesn't really clarify doing an extra check.

Co-authored-by: Nanush7 <59543882+Nanush7@users.noreply.github.com>

CarterKoala committed:
Update wca-guidelines.md

CarterKoala commented on wca-guidelines.md:
```diff
@@ -93,6 +93,7 @@ To be more informative, each Guideline is classified using one of the following
 - 4d++) [ADDITION] It is permitted for the puzzle to change its orientation when it is moved from the scrambler to the solving station, as long as no one is attempting to influence the randomness of the orientation (see [Regulation A2e1](regulations:regulation:A2e1)).
 - 4f+) [RECOMMENDATION] The WCA Delegate should generate sufficient scramble sequences for the entire competition ahead of time, including spare scramble sequences for extra attempts.
 - 4f++) [REMINDER] If the WCA Delegate generates any additional scramble sequences during the competition, the scramble sequences must be saved.
+- 4g+) [RECOMMENDATION] For Multi-Blind Solving, the scrambler should perform an extra check after scrambling to verify there are no duplicate scrambles.
```
```suggestion
- 4g+) [RECOMMENDATION] For Multi-Blind Solving, after scrambling all the cubes, the scrambler should verify there are no duplicate scrambles.
```

Nanush7 replied:
LGTM.

CarterKoala reviewed (commented):
I don't like using check as a noun here, I think this is better.

Nevseros committed:
Update wording

Co-authored-by: Carter Kucala <42009313+CarterKoala@users.noreply.github.com>

Rouxles commented:
This one will need a review from somebody

Nevseros commented on wca-guidelines.md:
```diff
@@ -93,6 +93,7 @@ To be more informative, each Guideline is classified using one of the following
 - 4d++) [ADDITION] It is permitted for the puzzle to change its orientation when it is moved from the scrambler to the solving station, as long as no one is attempting to influence the randomness of the orientation (see [Regulation A2e1](regulations:regulation:A2e1)).
 - 4f+) [RECOMMENDATION] The WCA Delegate should generate sufficient scramble sequences for the entire competition ahead of time, including spare scramble sequences for extra attempts.
 - 4f++) [REMINDER] If the WCA Delegate generates any additional scramble sequences during the competition, the scramble sequences must be saved.
+- 4g+) [RECOMMENDATION] For Multi-Blind Solving, after scrambling all the cubes, the scrambler should verify there are no duplicate scrambles.
```
```suggestion
- 4g+) [RECOMMENDATION] For Multi-Blind Solving, after scrambling all the cubes, the scrambler must verify there are no duplicate scrambles.
```

Nevseros committed:
Change `should` to `must`

Nevseros reviewed (approved):
LGTM!

Samuel-Baird commented on wca-guidelines.md:
```diff
@@ -93,6 +93,7 @@ To be more informative, each Guideline is classified using one of the following
 - 4d++) [ADDITION] It is permitted for the puzzle to change its orientation when it is moved from the scrambler to the solving station, as long as no one is attempting to influence the randomness of the orientation (see [Regulation A2e1](regulations:regulation:A2e1)).
 - 4f+) [RECOMMENDATION] The WCA Delegate should generate sufficient scramble sequences for the entire competition ahead of time, including spare scramble sequences for extra attempts.
 - 4f++) [REMINDER] If the WCA Delegate generates any additional scramble sequences during the competition, the scramble sequences must be saved.
+- 4g+) [RECOMMENDATION] For Multi-Blind Solving, after scrambling all the cubes, the scrambler must verify there are no duplicate scrambles.
```
it seems a bit odd to list a "must regulation" as a "recommendation". Perhaps "clarification" would be better suited?

CarterKoala replied:
Indeed, this should not be a recommendation. It will be fixed :)
